org.adempiere.base/src/org/compiere/util/WebUtil.java

```diff
@@ -460,38 +451,24 @@ public static void createResponse (HttpServletRequest request, HttpServletRespon
 				cookie.setMaxAge(2592000);      //  30 days in seconds   60*60*24*30
 			response.addCookie(cookie);
 		}
-		//  add diagnostics

```

Comment:
why to remove this, could be useful for debugging purposes - removing this makes the debug parameter useless